pkg/network/ebpf/c/ip.h

```diff
@@ -189,11 +189,25 @@ __maybe_unused static __always_inline void flip_tuple(conn_tuple_t *t) {
 // On older kernels, clang can generate Wunused-function warnings on static inline functions defined in
 // header files, even if they are later used in source files. __maybe_unused prevents that issue
 __maybe_unused static __always_inline void print_ip(u64 ip_h, u64 ip_l, u16 port, u32 metadata) {
+// support for %pI4 and %pI6 added in https://github.com/torvalds/linux/commit/d9c9e4db186ab4d81f84e6f22b225d333b9424e3
+#if defined(LINUX_VERSION_CODE) && LINUX_VERSION_CODE >= KERNEL_VERSION(5, 13, 0)
```

Comment:
This will be true for prebuilt programs, if the kernel used to compile is 5.13+. It could then run on a kernel < 5.13. You might need to use some of the compile mode defines to protect this.